rust/release/src/publisher.rs

```diff
@@ -748,6 +699,68 @@ fn docker_manifest_rm(target: &str) -> anyhow::Result<()> {
     Ok(())
 }
 
+// NOTE(duarte): The manifest create & push mechanism is a way of
+// "promoting images to production", by creating a manifest and pushing it under the
+// the name of another image, Docker will essentially just re-use the underlying images
+// to create a new one, under a new namespace. This is *crucial* for us because we
+// check if Docker releases are missing using the manifests.
+fn docker_manifest_push(target: &str) -> anyhow::Result<()> {
+    let args = ["manifest", "push", target];
+    log::debug!("running docker {:?}", args);
+    let cmd = Command::new("docker")
+        .args(args)
+        .output()
+        .context(format!("running docker {:?}", args))?;
+    anyhow::ensure!(
+        cmd.status.success(),
+        "Error running `docker {:?}` for {}\nstdout: {}\nstderr: {}",
+        args,
+        target,
+        String::from_utf8(cmd.stdout)?,
+        String::from_utf8(cmd.stderr)?
+    );
+    Ok(())
+}
+
+fn docker_manifest_create(
+    source_manifest: &DockerInspectResponse,
+    registry: &str,
+    source_repository: &str,
+    target_repository: &str,
+    tag: &str,
+) -> anyhow::Result<()> {
+    let target = format!("{}/{}:{}", registry, target_repository, tag);
+    // Make sure the manifest is removed locally, otherwise we can't
+    // (re)create it
+    docker_manifest_rm(&target)?;
+
+    // Now the actual manifest creation
+    let args = vec!["manifest".to_string(), "create".to_string(), target.clone()]
```

Comment:
I’d leave off the `vec!` as an array should work as well, right?

Reply:
Not in place (otherwise Clippy would've complained). 
From my understanding:
- The array's `into_iter` is just a call to `iter`
- Meaning it doesn't take ownership of the contents of the iterator 
- Thus it needs to be chained with `&String`s instead of `String`s
- These are dropped because they're local references 
Meaning it's a PITA to avoid one `vec!`